resolver/ecs_resolver.go

```diff
@@ -49,23 +50,26 @@ func NewECSResolver(cfg config.ECS) ChainedResolver {
 // and sets the client IP from the EDNS0 option to the request if this option is enabled
 func (r *ECSResolver) Resolve(ctx context.Context, request *model.Request) (*model.Response, error) {
 	if r.cfg.IsEnabled() {
+		ctx, logger := r.log(ctx)
+		_ = ctx
```

Comment:
Yep exactly. It complains it's unused otherwise, but I think it's better to keep it so if in the future we reference `ctx` we'll get the right one. Otherwise we'd need to notice that in `_, logger := r.log(ctx)` `_` is a context and update that code, which seems unlikely.

Reply:
Agreed, I was just curious if my guess was right since I couldn't test it right now. 😅

Reply:
Thanks for taking a look! :purple_heart: